Proxim Radio/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "dang",
+        "id": "474169687287136256"
+    },
+    "service": "Proxim Radio",
+    "description": {
+        "en": "The radio for all your chill time needs."
+    },
+    "url": "proximradio.com",
+    "version": "1.0",
+    "logo": "https://imgur.com/a/5na9Irh",
+    "thumbnail": "https://imgur.com/a/XAoJDPA",

```

Comment:
Images need to be the .png version of it. (.png needs to be in the url)

Reply:
Fixed problem! c: